light_node/Cargo.toml

```diff
@@ -9,6 +9,7 @@ default-run = "light-node"
 clap = "2.33"
 failure = "0.1"
 futures = "0.3"
+itertools = "0.10"
```

Comment:
@mat13mn 
I dont see any usage of this in this module,
please remove if not needed